etsin_finder/frontend/locale/english.js

```diff
@@ -326,6 +326,10 @@ const english = {
       130: 'Transfer failed',
       140: 'Available'
     },
+    useDoiHeader: 'Creation of a DOI',
+    useDoiContent: 'You have selected DOI as primary identifier for your dataset instead of URN. DOI requires a defined issued date. A DOI (Digital Object Identifier) will be created and stored in the DataCite Service and it cannot be removed. Are you sure you want to select DOI?',
```

Comment:
Maybe add here that DOI requires also a publisher.

Reply:
Resolved by commit 1ffec4f